.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [pull_request]
+
+jobs:
+  ci:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Install dependencies
+        run: |
+          brew install doxygen
+          brew install graphviz
+
+      - name: Create google-services.json
+        env:
+          GOOGLE_SERVICES_JSON: ${{ secrets.GOOGLE_SERVICES_JSON }}
+        run: |
+          echo "$GOOGLE_SERVICES_JSON" > samples/android/Assets/google-services.json
+
+      - name: Build
```

Comment:
we don't want to run the samples?

Reply:
`./gradlew build` does build the samples. It doesn't create the NuGet Packages, I think.

Reply:
build should cover all. the current ci is only building samples so this is doing more. Ill look into the nuget package parts